src/onegov/landsgemeinde/cli.py

```diff
@@ -0,0 +1,107 @@
+""" Provides commands used to initialize org websites. """
+import click
+
+from onegov.core.cli import command_group, pass_group_context, abort
+from onegov.core import html
+from onegov.org.models import Organisation
+from onegov.landsgemeinde.collections import VotumCollection
+from onegov.landsgemeinde.collections import AgendaItemCollection
+from onegov.landsgemeinde.models import AgendaItem
+
+cli = command_group()
+
+
+@cli.command(context_settings={'creates_path': True})
+@click.argument('name')
+@click.option(
+    '--locale',
+    default='de_CH',
+    type=click.Choice(['de_CH', 'fr_CH', 'it_CH'])
+)
+@pass_group_context
+def add(group_context, name, locale):
+    """ Adds an org with the given name to the database. For example:
+
+        onegov-org --select '/onegov_org/evilcorp' add "Evilcorp"
+
+    """
+
+    def add_org(request, app):
+
+        if app.session().query(Organisation).first():
+            abort("{} already contains an organisation".format(
+                group_context.selector))
+
+        app.settings.org.create_new_organisation(app, name, locale=locale)
+
+        click.echo("{} was created successfully".format(name))
+
+    return add_org
+
+
```

Comment:
```suggestion
```
You don't need to duplicate this command from `onegov.org` to `onegov.landsgemeinde`, since you inherit from `TownsApp` the CLI command should work just fine, you just need to use a landsgemeinde specific selector.

Reply:
Thanks for noticing. I actually merged this issue from another (unmerged) branch, this command will probably never be used, so I'm going to remove it from here again.